security/encrypt-cluster-pod-traffic.md

```diff
@@ -36,10 +36,94 @@ This how-to guide uses the following {{site.prodname}} features:
 
 1. Install WireGuard on cluster nodes using {% include open-new-window.html text='instructions for your operating system' url='https://www.wireguard.com/install/' %}. Note that you may need to reboot your nodes after installing WireGuard to make the kernel modules available on your system.
```

Comment:
To make this easier with numbering, I suggest making this just a paragraph.

Reply:
I have removed the numbering in that section altogether. This also helps with numbered instructions in OpenShift mixing with the common instructions. PTAL.